packages/elemental-theme/src/custom-elements/ef-clock.less

```diff
@@ -74,10 +74,11 @@
     }
   }
 
-  [part~='hours']:focus, [part~='minutes']:focus, [part~='seconds']:focus {
-    color: @scheme-color-primary;
-
-    [part='increment-button'], [part='decrement-button'] {
+  &:focus {
+    [part~='hours'], [part~='minutes'], [part~='seconds'] {
+      color: @scheme-color-primary;
+    }
+    [active]{
```

Comment:
Wrong formatting, space is missing after ].

Why do you need it for focus? Can active be always set to `block`?